astrodb_utils/utils.py

```diff
@@ -30,59 +30,129 @@ class AstroDBError(Exception):
 
 
 def load_astrodb(
-    db_file,
+    db_path,
     data_path="data/",
     recreatedb=True,
-    reference_tables=[
-        "Publications",
-        "Telescopes",
-        "Instruments",
-        "Versions",
-        "PhotometryFilters",
-        "Regimes",
-        "AssociationList",
-        "ParameterList",
-        "CompanionList",
-        "SourceTypeList",
-    ],
+    reference_tables=None,
     felis_schema=None
 ):
     """Utility function to load the database
 
     Parameters
     ----------
-    db_file : str
-        Name of SQLite file to use
+    db_path : str
+        Path to the directory containing the __init__.py and schema.yaml file for the database
+        This name is used to name the database file, e.g. 'stars' will create 'stars.sqlite'
     data_path : str
         Path to data directory; default 'data/'
     recreatedb : bool
         Flag whether or not the database file should be recreated
     reference_tables : list
         List of tables to consider as reference tables.
         Default: Publications, Telescopes, Instruments, Versions, PhotometryFilters
+        Looks in <db_name>/__init__.py for the list of tables.
     felis_schema : str
         Path to Felis schema; default None
+        Looks in db_name/schema.yaml for the schema path.
+
+    Returns
+    -------
+    db : Astrodbkit Database object
+
+    Also creates the database file if it does not exist.
+    If recreatedb is True, it removes the existing database file before creating a new one.
     """
 
-    db_file_path = Path(db_file)
+    db_name = os.path.basename(db_path)
+
+    db_file = db_name + ".sqlite"
     db_connection_string = "sqlite:///" + db_file
+    logger.debug(f"Database connection string: {db_connection_string}")
+
+    # Load the reference tables from the db_name module if not provided
+    if reference_tables is None:
+        try:
+            init_path = os.path.join(db_path, "__init__.py")
+            spec = importlib.util.spec_from_file_location(db_name, init_path)
+            db_module = importlib.util.module_from_spec(spec)
+            spec.loader.exec_module(db_module)
+            REFERENCE_TABLES = db_module.REFERENCE_TABLES
```

Comment:
We need to add some message (logger.info or even warning) that alerts the user this was one on their behalf.